src/auth/service/auth.service.ts

```diff
@@ -0,0 +1,78 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+import { compare } from 'bcrypt';
+import { CustomersService } from '../../customers/service/customers.service';
+import { User } from '../domain/user.domain';
+
+@Injectable()
+export class AuthService {
```

Comment:
This service contains both methods used for `jwt-auth` and `local-auth`.  One idea could be to split this service into 2, then it would follow the separation we have for the 2 auth types (2 strategies, 2 guards).


But do not change anything now. Your approach is fine.

Reply:
done